public/app/core/components/RolePicker/api.ts

```diff
@@ -19,10 +22,11 @@ export const fetchUserRoles = async (userId: number, orgId?: number): Promise<Ro
     userRolesUrl += `?targetOrgId=${orgId}`;
   }
   try {
-    const roles = await getBackendSrv().get(userRolesUrl);
+    let roles = await getBackendSrv().get(userRolesUrl);
     if (!roles || !roles.length) {
       return [];
     }
+    roles = roles.map(addDisplayNameForFixedRole);
```

Comment:
We also need to update `fetchTeamRoles()`

Reply:
[3eacfae](https://github.com/grafana/grafana/pull/62845/commits/3eacfae4566a50e4bc1d9939093976868018cba2)